docker-compose.yml

```diff
@@ -1,349 +1,383 @@
-#UDeploy server (required) - Can provide multiple
-  - UDEPLOY_URL:-http://udeploy.company.com
-#UDeploy user name (required)
-  - UDEPLOY_USERNAME:-bobama
-#UDeploy password (required)
-  - UDEPLOY_PASSWORD:-s3cr3t
-#Collector schedule (required)
-#  - UDEPLOY_CRON:-0 0/5 * * * *
-
-hygieia-xldeploy-collector:
-  image: hygieia-xldeploy-collector:latest
-  container_name: hygieia-xldeploy
-  volumes:
-  - ./logs:/hygieia/logs
-  links:
-  - mongodb:mongo
-  - hygieia-api
-  environment:
-#XLDeploy server (required) - Can provide multiple
-  - XLDEPLOY_URL:-http://xldeploy.company.com
-#XLDeploy user name (required)
-  - XLDEPLOY_USERNAME:-bobama
-#XLDeploy password (required)
-  - XLDEPLOY_PASSWORD:-s3cr3t
-#Collector schedule (required)
-#  - XLDEPLOY_CRON:-0 0/5 * * * *
-
-hygieia-artifactory-artifact-collector:
-  image: hygieia-artifactory-artifact-collector:latest
-  container_name: hygieia-artifactory
-  volumes:
-  - ./logs:/hygieia/logs
-  links:
-  - hygieia-api
-  environment:
-#JFrog Artifactory server (required) - Can provide multiple 
-  - ARTIFACTORY_URL=https://www.jfrog.com
-# Matches maven artifacts of the form [org]/[module]/[version]/[module]-[version]([-classifier])(.[ext])
-  - ARTIFACTORY_PATTERN0=(?<group>.+)/(?<module>[^/]+)/(?<version>[^/]+)/(?<artifact>\\k<module>)-\\k<version>(-(?<classifier>[^\\.]+))?(\\.(?<ext>.+))?
-# Matches ivy files of the form [org]/[module]/[revision]/ivy-[revision](-[classifier]).xml 
-  - ARTIFACTORY_PATTERN1=(?<group>.+)/(?<module>[^/]+)/(?<version>[^/]+)/(?<artifact>ivy)-\\k<version>(-(?<classifier>[^\\.]+))?\\.(?<ext>xml)
-# Matches ivy artifact files of the form [org]/[module]/[revision]/[type]/[artifact]-[revision](-[classifier])(.[ext])
-  - ARTIFACTORY_PATTERN2=(?<group>.+)/(?<module>[^/]+)/(?<version>[^/]+)/(?<type>[^/]+)/(?<artifact>[^\\.-/]+)-\\k<version>(-(?<classifier>[^\\.]+))?(\\.(?<ext>.+))?
-
-  
-# Artifactory user name (required)
-#  - ARTIFACTORY_USERNAME=bobama
-# Artifactory API key (required)
-#  - ARTIFACTORY_API_KEY=s3cr3t
-#  - ARTIFACTORY_CRON=0 0/1 * * * *
-# The repo to collect artifacts from (required) - Can provide multiple (comma separated for each server)
-#  - ARTIFACTORY_REPO=prerelease,release
-# Artifactory REST endpoint
-#  - ARTIFACTORY_ENDPOINT=artifactory/
-
-hygieia-appdynamics-collector:
-  image: hygieia-appdynamics-collector:latest
-  container_name: hygieia-appdynamics
-  volumes:
-  - ./logs:/hygieia/logs
-  links:
-  - mongodb:mongo
-  - hygieia-api
-# place any values you want to change in docker-compose.override.yml
-#  environment:
-#  - APPDYNAMICS_INSTANCE_URL=http://appdynamics
-#  - APPDYNAMICS_USERNAME=username
-#  - APPDYNAMICS_PASSWORD=password
-#  - APPDYNAMICS_DASHBOARD_URL=http://appdynamics/controller/#/location=APP_DASHBOARD&timeRange=last_15_minutes.BEFORE_NOW.-1.-1.15&application=%s&dashboardMode=force
-#  - APPDYNAMICS_CRON='-1 * * * * *'
+version: '2'
+volumes:
+  data: {}
+services:
+  mongodb:
+    image: mongo:latest
```

Comment:
If I remember correctly this was my core change. Using the standard mongo image didn't work for me, so I had to change that.

Reply:
Can anyone confirm using mongo:latest now properly works for them? If so, I'll give it another try.

Reply:
Can't confirm. As of right now I haven't been able to get the db-setup to work, but haven't spent much time with it. I'm just using a separate mongo instance with a persistent data volume. Which mongo image did you use to get it to work?